test/run.sh

```diff
@@ -24,10 +24,11 @@ Options:
   -v DIR             The venv directory. (${RF_VENV})
   -i PATH            The variables file. (${RF_VARIABLES})
   -s NAME=VALUE      To enable an stress condition.
+  -k SKIP_TESTS      List of Tests to skip.
```

Comment:
Nit: we should probably say: Comma separated list of tests to skip

Reply:
Ah, and while you're there, please also fix the above comment.
`an stress` -> `a stress`